pkg/network/netlink/conntracker.go

```diff
@@ -174,6 +188,22 @@ func (ctr *realConntracker) GetTranslationForConn(c network.ConnectionStats) *ne
 	return t.IPTranslation
 }
 
+func (ctr *realConntracker) RefreshTelemetry() {
+	for {
+		ctr.RLock()
+		ctr.stats.cacheSize.Set(float64(ctr.cache.cache.Len()))
+		ctr.stats.orphanSize.Set(float64(ctr.cache.orphans.Len()))
+		ctr.RUnlock()
+		ctr.consumer.GetStats()
```

Comment:
What is this for?

Reply:
I believe this was debug that was removed